spec/features/tests/user_view_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+feature 'user signs out', %Q{
+  As a user
+  I want to view a list of food trucks
+  so that I can decide where to eat
+} do
+  # Acceptance Criteria
+  # * If a user goes to the index page, they see a list of food trucks.
+
+  scenario 'user views index page' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.